frontend/stmt_inferrer.py

```diff
@@ -340,6 +341,46 @@ def _infer_class_def(node, context, solver):
     context.set_type(node.name, result_type)
 
 
+def _infer_import(node, context, solver):
+    """Infer the imported module in normal import statement"""
+    import_handler = ImportHandler()
```

Comment:
Doesn't the ImportHandler only have static methods? Then you wouldn't need to create an instance, right?